sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/trees/TreeNodeSuite.scala

```diff
@@ -871,28 +871,6 @@ class TreeNodeSuite extends SparkFunSuite with SQLHelper {
     }
   }
 
-  test("SPARK-32999: TreeNode.nodeName should not throw malformed class name error") {
-    val testTriggersExpectedError = try {
-      classOf[MalformedClassObject.MalformedNameExpression].getSimpleName
```

Comment:
Can MalformedClassObject be removed safely?

Reply:
you're awesome 👍